README.md

```diff
@@ -1,24 +1,44 @@
-# Frobnicator (this is a template)
+[![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-22041afd0340ce965d47ae6ef1cefeee28c7c493a6346c4f15d667ab976d596c.svg)](https://classroom.github.com/a/M0kyOMLZ)
+
+# Stock Market Data Analysis Application (this is a template)
 
 ## Authors
-- Andrzej Głuszak (@agluszak on GitHub)
-- Linus Torvalds (@torvalds on GitHub)
+- Antoni Rajtar (@ar448467 on GitHub)
 
 ## Description
-Frobnicator is going to be a platformer game similar to Super Mario Bros made using Korge game engine.
+I will create a desktop application with the following functionalities:
```

Comment:
czy aplikacja będzie podzielona na klienta i serwer, czy też wszystkie dane będą przechowywane lokalnie? jeśli to drugie, to w jaki sposób?